tests/integration/api_v3/http/posStakes.test.js

```diff
@@ -31,47 +31,40 @@ const endpoint = `${baseUrlV3}/pos/stakes`;
 
 describe('Stakes API', () => {
 	let refValidator;
+	let maxNumberSentStakes;
 	beforeAll(async () => {
-		let response;
+		const posConstants = await api.get(`${baseUrlV3}/pos/constants`);
+		maxNumberSentStakes = posConstants.data.maxNumberSentStakes;
+
 		do {
 			// eslint-disable-next-line no-await-in-loop
-			response = await api.get(`${baseUrlV3}/dpos/delegates?limit=1`);
-		} while (!response.data);
-		[refValidator] = response.data;
+			const { data: [stakeTx] = [] } = await api.get(`${baseUrlV3}/transactions?moduleCommand=pos:stake&limit=1`);
+			if (stakeTx) {
+				refValidator = stakeTx.sender;
+			}
```

Comment:
Staker is the sender of a stake transaction and validator is the recipient, please update all references in both tests. Also, the if condition will always pass as it is an array.

```suggestion
			if (stakeTx.length) {
				refStaker = stakeTx.params. recipient;
			}
```

Reply:
We are already extracting transaction object. So, `stakeTx` is an object here.